src/core_plugins/kibana/common/tutorials/tutorial_schema.js

```diff
@@ -28,7 +28,7 @@ const artifactsSchema = Joi.object({
 const instructionSchema = Joi.object({
   title: Joi.string(),
   textPre: Joi.string(),
-  commands: Joi.array().items(Joi.string()),
+  commands: Joi.array().items(Joi.string().allow('')),
```

Comment:
@nreese You ok with this? It will allow something like:

```js
commands: ['a', '', 'b']
```

which will render:
```
a

b
```